nf-core/modules
# Modify DAS_Tool input structure, bump version, switch to nf-test

1. Old version of dastool/dastool module took the proteins fasta on a separate input channel, but this has to match the required contigs input so I've moved the protein input to the same channel.
2. Bump to latest DAS_Tool version
3. Relabel two mislabelled output channels in dastool/dastool
4. Change protein fasta decompression to check if input proteins are compressed in dastool/dastool
5. Switch old tests to new nf-tests for all dastool modules

## PR checklist

Closes #XXX 

- [ ] If you've added a new tool - have you followed the module conventions in the [contribution docs](https://github.com/nf-core/modules/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, include test data in your PR.
- [ ] Remove all TODO statements.
- [x] Emit the `versions.yml` file.
- [ ] Follow the naming conventions.
- [ ] Follow the parameters requirements.
- [ ] Follow the input/output options guidelines.
- [ ] Add a resource `label`
- [x] Use BioConda and BioContainers if possible to fulfil software requirements.
- Ensure that the test works with either Docker / Singularity. Conda CI tests can be quite flaky:
  - For modules:
    - [x] `nf-core modules test <MODULE> --profile docker`
    - [ ] `nf-core modules test <MODULE> --profile singularity`
    - [ ] `nf-core modules test <MODULE> --profile conda`
  - For subworkflows:
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile docker`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile singularity`
    - [ ] `nf-core subworkflows test <SUBWORKFLOW> --profile conda`
